Updated FAQ Content

neeru24 opened the pull request:
I have updated the FAQ section to remove previously repeated questions and added some insightful new content. This should enhance the user experience by providing clearer and more diverse information.

Kindly merge it @ajaynegi45 and let me know if any changes are needed. 

fixes: #164

ajaynegi45 commented:
> I have updated the FAQ section to remove previously repeated questions and added some insightful new content. This should enhance the user experience by providing clearer and more diverse information.
> 
> Kindly merge it @ajaynegi45 and let me know if any changes are needed.
> 
> fixes: #164

Still last two question is repeating.

neeru24 commented:
> > I have updated the FAQ section to remove previously repeated questions and added some insightful new content. This should enhance the user experience by providing clearer and more diverse information.
> > 
> > Kindly merge it @ajaynegi45 and let me know if any changes are needed.
> > 
> > fixes: #164
> 
> Still last two question is repeating.


You could have told me, I'll change that. By mistake,that got repeated. If possible please reopen this issue. @ajaynegi45
